src/components/Navbar.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { AiOutlineLeft } from 'react-icons/ai';
+import { BsFillGearFill } from 'react-icons/bs';
+import { Link } from 'react-router-dom';
+import './styles/Navbar.css';
+
+const Navbar = () => {
+  return (
+    <nav className="navbar-container">
+      <div>
+        <div>
+          <ul className="navbar">
+            <li>
+              <Link to="/" className="home"> <AiOutlineLeft /> </Link>
```

Comment:
- [ ] This _back_ (_to homepage_) navigation button is not working as intended when clicked. It's **unable to navigate a user back to the homepage** when clicked. Kindly ensure to improve upon this aspect.

Reply:
This button is actually working. Maybe you mean when doing the search with the input? That does not work because you're still on the same page.